dmscripts/notify_buyers_to_award_closed_briefs.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, date, timedelta
+
+import dmapiclient
+from dmutils.email import DMNotifyClient
+from dmutils.email.exceptions import EmailError
+from dmutils.formats import DATE_FORMAT
+
+from dmscripts.helpers import logging_helpers, date_helpers, brief_data_helpers
+
+logger = logging_helpers.configure_logger({'dmapiclient': logging_helpers.logging.INFO})
+
+
+def _filter_brief_users_by_optional_user_id_list(brief, user_id_list):
```

Comment:
This could also go on one line with a python ternary like so:
```return filter(lambda user: user['id'] in user_id_list, brief['users']) if user_id_list else brief['users']```

Reply:
I think some readability might be lost here? As it is, it's pretty clear that we apply the filter if the user id list is given, otherwise we return the unchanged list.